sklearn/pipeline.py

```diff
@@ -25,7 +26,24 @@
 __all__ = ['Pipeline', 'FeatureUnion']
 
 
+def _pretty_print(step_info):
+    """Helper method to print the information about execution of a particular
+    step of Pipeline / FeatureUnion (if verbosity is enabled). It receives a
+    string having information about current step and prints it in such a way
+    that its length is 70 characters.
+
+    Parameters
+    ----------
+    step_info : str
```

Comment:
What's the code in grid search you're talking about? The only logging is within joblib, right?